tests/codebuild/run_integtest.sh

```diff
@@ -1,33 +1,58 @@
 #!/bin/bash
 
-source tests/codebuild/common.sh
-
 # TODOs
 # 1. Add validation for each steps and abort the test if steps fails
 # Build environment `Docker image` has all prerequisite setup and credentials are being passed using AWS system manager
+# crd_namespace is currently hardcoded here for cleanup. Should be an env variable set in the pipeline and .env file
+
+source tests/codebuild/common.sh
+crd_namespace="test-namespace"
```

Comment:
Can we make this overridable using an env variable, even if we don't actually override it in the pipeline. 

`crd_namespace=${NAMESPACE_OVERRIDE:-"test-namespace"}`